editing/edit-features-with-feature-linked-annotation/src/main/java/com.esri.samples.edit_features_with_feature_linked_annotation/EditFeaturesWithFeatureLinkedAnnotationSample.java

```diff
@@ -69,13 +71,20 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create the map with a light gray canvas basemap centered on Loudoun, Virginia
-      ArcGISMap map = new ArcGISMap(Basemap.Type.LIGHT_GRAY_CANVAS_VECTOR, 39.0204, -77.4159, 18);
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with a basemap style
```

Comment:
to be consistent with the comment style used for topographic and streets basemap, we could use the following comment unless there was a reason to keep it generic? this change would apply across a number of these classes if you want to go for it. 

```suggestion
      // create a map with the light gray basemap style
```

Reply:
For reference, we are going with `create a map with the light gray basemap style` for whatever basemap style is being set.